backend/src/auth/auth.controller.ts

```diff
@@ -145,6 +166,16 @@ export class AuthController {
   @PublicRoute()
   @Post("/reset-password")
   @HttpCode(HttpStatus.OK)
+  @ApiOperation({ summary: API_OPERATIONS.RESET_PASSWORD_SUMMARY})
+  @ApiOkResponse({
+    description:
+      API_RESPONSES_DESCRIPTION
+      .SUCCESSFUL_RESET_PASSWORD_DESCRIPTION,
+  })
+  @ApiBadRequestResponse({
+    description: API_RESPONSES_DESCRIPTION
+    .BAD_REQUEST_INVALID_CREDENTIALS_DESCRIPTION,
+  })
```

Comment:
i don’t think all bad requests is because of invalid credentials - maybe can put smth like “server failed to complete request?” for cases where some issue occurred in server like a prisma error or smth